types/heremaps/index.d.ts

```diff
@@ -5681,7 +5694,7 @@ declare namespace H {
              * @param opt_locale {(H.ui.i18n.Localization | string)=} - the language to use (or a full localization object).
              * @returns {H.ui.UI} - the UI instance configured with the default controls
              */
-            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes, opt_locale?: H.ui.i18n.Localization | string): UI;
+            static createDefault(map: H.Map, mapTypes: H.service.MapType | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
```

Comment:
This again is a case where the documentation just states `{Object<H.service.MapType>}` as the type of the parameter. I tested it out and asked HERE for clarification, that type basically means this in TypeScript:

```ts
interface MapTypes {
  [index: string]: H.service.MapType;
}
```

The other line you commented on @rbuckton has since seen that type replaced with `H.service.DefaultLayers`. This remains the same, so I *think* the definition should stay as it is. Hopefully with my two comments I can show you where the confusion comes from the docs.